rclcpp/test/rclcpp/test_publisher_with_type_adapter.cpp

```diff
@@ -161,6 +125,8 @@ TEST_F(TestPublisher, various_creation_signatures) {
       auto publisher = node->create_publisher<StringTypeAdapter>("topic", 42);
       (void)publisher;
     }
+
+    node.reset();
```

Comment:
nit, I think the shared_ptr `node` is being destructed at the end of this block anyways, so this line can be removed.

Reply:
Yeah, good point.  I probably took that idiom from somewhere else, but probably not necessary here.  I'll remove it.

Reply:
Fixed in the latest (I ended up squash-merging this into the appropriate commit).